plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -441,12 +442,16 @@ public List<String> getAllTables(String databaseName)
     @Override
     public synchronized List<String> getTablesWithParameter(String databaseName, String parameterKey, String parameterValue)
     {
-        // TODO
-        throw new UnsupportedOperationException("getTablesWithParameter for GlueHiveMetastore is not implemented");
+        return getAllViews(databaseName, table -> parameterValue.equals(table.getParameters().get(parameterKey)));
```

Comment:
`Table#getParameters` is nullable. It would be better to use `firstNonNull(table.getParameters(), ImmutableMap.of())` just in case.  Relates to https://github.com/trinodb/trino/commit/61c1b5a38734936a9c1947c9fc871c5877c7821d